src/app/components/BibPage/BibDetails.jsx

```diff
@@ -301,8 +294,6 @@ const BibDetails = (props) => {
   }) => {
     const onClick = (event) => {
       event.preventDefault();
-
-      trackDiscovery('Bib fields', `${analyticsLabel} - ${analyticsValue}`);
```

Comment:
This function no longer uses `analyticsLabel` or `analyticsValue`. Those should be removed from the function argument object and also the parameters passed wherever `searchRouterLink` is being called.

Reply:
Good catch, removed those values as well as the onClick handlers in the Request Buttons